src/devices/lincukoo.ts

```diff
@@ -60,21 +62,25 @@ export const definitions: DefinitionWithExtend[] = [
                 .withValueMax(90)
                 .withValueStep(1)
                 .withDescription("Sensitivity of the radar"),
-            e.numeric("fading_time", ea.STATE_SET).withValueMin(3).withValueMax(1799).withValueStep(1).withDescription("Fading time").withUnit("s"),
-            e.binary("relay_switch", ea.STATE_SET, "ON", "OFF").withDescription("Relay switch"),
-            e.binary("radar_switch", ea.STATE_SET, "ON", "OFF").withDescription("Radar switch"),
+            e.numeric("Fading_time", ea.STATE_SET).withValueMin(3).withValueMax(1799).withValueStep(1).withDescription("Fading time").withUnit("s"),
```

Comment:
```suggestion
            e.numeric("fading_time", ea.STATE_SET).withValueMin(3).withValueMax(1799).withValueStep(1).withDescription("Fading time").withUnit("s"),
```

Use `snake_case` (same goes for the others)